src/rules/javascript/webextension-unsupported-api.js

```diff
@@ -0,0 +1,21 @@
+import { UNSUPPORTED_API } from 'messages/javascript';
+import { hasBrowserApi } from 'schema/browser-apis';
+
+export default {
+  create(context) {
+    return {
+      MemberExpression: function(node) {
+        if (node.object.object &&
+            ['chrome', 'browser'].includes(node.object.object.name)) {
+          let namespace = node.object.property.name;
```

Comment:
We had this code in the [webextension-api.js](https://github.com/mozilla/addons-linter/blob/e3c310e8efdb40dc50d96617f5c6708cbef1d786/src/rules/javascript/webextension-api.js#L10) before and it was fine.

Since this is a MemberExpression I think this should always be available.